### PR TITLE
Create ChainAegis.md

### PR DESCRIPTION
**Pull request by ikaros-lucifer**

### CODE REVIEW

File: help-docs/marketplace/apps/ChainAegis.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+3. Your community members can check whether the address is risk through _/chainaegis check_, and they can also report addresses and risk details through _/chainaegis report_.
+
+4. Your community members can apply for other security services such as smart contract auditing, on-chain analysis, asset tracking, data apis through _/chainaegis request_
+
+**Commands**
+
+| **Command** | **Description** |
+| --- | --- |
+| /chainaegis alert | Get the latest 5 Rugpull and 5 security incidents. |
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/ChainAegis.md:19]: wrap in \` please to make explicit for each command

eg. /chainaegis alert --> \`/chainaegis alert\` renders as `/chainaegis alert`

File: help-docs/marketplace/apps/ChainAegis.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+| /chainaegis request on-chain-analysis | Apply for on-chain analysis service and we will contact you upon receipt. |
+| /chainaegis request kyt | Apply for ChainAegis KYT and we will contact you upon receipt. |
+| /chainaegis request fund-track | Apply for ChainAegis fund track service and we will contact you upon receipt. |
+| /chainaegis request data-api | Apply for ChainAegis data APIs and we will contact you upon receipt. |
+
+**Support**
+
+- Twitter: https://twitter.com/ChainAegis
+- Email: sharkteam@payegis.com
+- Discord: https://discord.com/invite/jGH9xXCjDZ
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/ChainAegis.md:32]: make sure this discord invite will not expire. default invites expire after 7 days.

File: help-docs/marketplace/apps/ChainAegis.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+**About this app:**
+
+ChainAegis is an AI-driven risk assessment and alerts platform. It provides rugpull alerts, security incident alerts, and you can check an address' reputation and risk types. ChainAegis also provides other security services such as smart contract auditing, on-chain analysis, asset tracking, and data APIs.
+
+**To get started:**
+
+1. Install the ChainAegis Miniapp from the [Collab.Land Marketplace](../getting-started)
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/ChainAegis.md:7]: There is no additional registration?

File: help-docs/marketplace/apps/ChainAegis.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+2. Your community members will get the latest 5 Rugpull and 5 security incidents through _/chainaegis alert_.
```
Comment (iSpeakNerd) [inline help-docs/marketplace/apps/ChainAegis.md:9]: rugpull and security incidents from where and across what chains?

Comment (iSpeakNerd) [reply help-docs/marketplace/apps/ChainAegis.md:9]: ### Invert all directions to active steps:
e.g. Do X to achieve Y.

> 2. Your community members will get the latest 5 Rugpull and 5 security incidents through _/chainaegis alert_.

becomes
> 2. Use `/chainaegis alert` to pull the latest 5 Rugpull and 5 security incidents.

**Review by iSpeakNerd — CHANGES_REQUESTED** [review]
- [ ] Invert all directions to active steps:
e.g. Do X to achieve Y.
- [ ] wrap all commands in backtics \` to indicate exact code command
e.g. \`/chainaegis alert\` will render as `/chainaegis alert`
- [ ] make sure Discord invite [will not expire](https://support.discord.com/hc/en-us/articles/208866998-Invites-101#h_01H2DMMG355MV7GJDQ9F65SDVG), default invites expire 7 days from creation

File: help-docs/marketplace/apps/ChainAegis.md (hunk 3 above)
Comment (ikaros-lucifer) [reply help-docs/marketplace/apps/ChainAegis.md:7]: Yes, no additional registration here.

File: help-docs/marketplace/apps/ChainAegis.md (hunk 4 above)
Comment (ikaros-lucifer) [reply help-docs/marketplace/apps/ChainAegis.md]: OK, I'll modify all directions to active steps

File: help-docs/marketplace/apps/ChainAegis.md (hunk 1 above)
Comment (ikaros-lucifer) [reply help-docs/marketplace/apps/ChainAegis.md]: I'll warp all commands in `

File: help-docs/marketplace/apps/ChainAegis.md (hunk 2 above)
Comment (ikaros-lucifer) [reply help-docs/marketplace/apps/ChainAegis.md]: I've update new discord invite, thanks for reminder.

**Review by iSpeakNerd — APPROVED** [review]

**Pull request merged**